src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -387,6 +414,20 @@ public IEnumerable<TestMethod> BuildFrom(IMethodInfo method, Test suite)
         {
             TestMethod test = new NUnitTestCaseBuilder().BuildTestMethod(method, suite, GetParametersForTestCase(method));
 
+            if (_untilDate.HasValue)
+            {
+                if (_untilDate > DateTimeOffset.UtcNow)
```

Comment:
Nitpick: `_untilDate.Value`

(Not necessary, but I'd write `if (_untilDate != null && _untilDate.Value > DateTimeOffset.UtcNow)`.)

Reply:
Very nitpick (even if it is the sort of thing I nitpick about) 😝